### ISSUE TITLE
Monitor wrongly notifies device disconnection and battery level

### ISSUE THREAD

**Issue by fnobilia** [RADAR-CNS/RADAR-Backend#24]
Received this email while the device was still connected.

```
The device {"userId": "KCLTest0", "sourceId": "00:07:80:1F:52:F3"} seems disconnected. It was last seen on Fri, 2 Jun at 12:56:47 UTC (300 seconds ago). If this is not intended, please ensure that it gets reconnected.
```

After this message no `reconnection` notification has been sent. Meanwhile, I received a sequence of these messages
```
The battery level of {"userId": "KCLTest0", "sourceId": "00:07:80:1F:52:F3"} has returned to normal. No further action is needed.
```

RAW data in CSV format is available if useful for troubleshooting.

### Update

Still receiving 
```
The battery level of {"userId": "KCLTest0", "sourceId": "00:07:80:1F:52:F3"} has returned to normal. No further action is needed.
```

after the device has been disconnected

**Comment by yatharthranjan** [comment]
@afolarin The issue already exists. I can look into solving this. What do you think?

**Comment by afolarin** [comment]
Yes please. There may already be a working implementation separately done by @sboettcher

**Comment by afolarin** [comment]
I think we can drop the recharged notification if this is the principle source of problematic repetitive emails. I don't think it is so important as there is a battery indicator already

**Comment by yatharthranjan** [comment]
@afolarin Yes I am looking into it. If nothing works then I will just remove the notification message about normal battery level.

**Comment by yatharthranjan** [comment]
Closed with #37
